src/screens/CardInfo/CardInfoScreen.tsx

```diff
@@ -0,0 +1,216 @@
+import React, { useState } from 'react';
+import { StyleSheet, Text, View, StatusBar, Pressable } from 'react-native';
+
+import useOnPressHandlers from '../../hooks/useOnPressHandlers';
+
+import StyleBase from '../../styles/StyleBase';
+
+import colors from '../../constants/colors';
+
+import { Benefit } from '../../types';
+
+import TopBar from '../../components/Bars/TopBar';
+import TapBar from '../../components/Bars/TapBar';
+import CardTile from '../../components/Cards/CardTile';
+import Tile from '../../components/Miscellaneous/Tile';
+import BenefitList from '../../components/Benefits/BenefitList';
+import CustomButton from '../../components/Miscellaneous/CustomButton';
+import BoxContainer from '../../components/Miscellaneous/BoxContainer';
+
+const SHOW_VIRTUAL = 1;
+
+//temp
+interface ICardInfoScreen {
+  navigation: any; //proper type
+  Card?: any; //todo
+  benefits: Benefit[];
+}
+
+export default function CardInfo({ navigation, Card, benefits }: ICardInfoScreen) {
+  /*
+  todo:b
+  check if virtual or real
+  needs some styling
+  tested on pixel 6 pro (1440x3120), components MAY NOT fit properly on other models
+  proper card icon
+  proper account balance from api
+  proper descriptions
+  proper everything (after api implementation)
+  */
+
+  const [screenState, setState] = useState(1);
+
+  const { onPressBack } = useOnPressHandlers();
+
+  return SHOW_VIRTUAL ? (
+    <View style={StyleBase.container}>
+      <StatusBar barStyle='default' />
+      <TopBar iconLeft={'arrow-left'} onPressLeft={() => onPressBack(navigation)} />
+      {/* todo: apply paddings
+                image as Card.businessDetails.iconImageId */}
+      {/*temp solution - Tile of CardTile style override */}
+      <View style={styles.cardTile}>
+        <CardTile
+          image={require('../../assets/images/biedronka_homepage.jpg')}
```

Comment:
Są już zamockowane karty z img, możesz przekazać to w propsie `card` (a jak przychodzimy tu z nawigacji, to nawigacja też może przekazywać paramsy - wiec gitez).

Reply:
Wtedy `card.image` i essa